Add-ons/Umbraco-Deploy/Installing-Deploy/New-site/index.md

```diff
@@ -144,7 +144,15 @@ The `type` value is for informational purposes in the backoffice but in most cas
 
 The URLs for each environment needs to be accessible by the other environments over **HTTPS**.
 
-:::note 
+When you have set up your environments in the UmbracoDeploy.Config the following AppSetting needs to be set on the different environments that you have set up with Umbraco Deploy:
+
+```c#
+<add key="Umbraco.Deploy.EnvironmentName" value="TheEnvironmentTypeHere" />
+```
+
+The value needs to be the **environment type** that have been set in the UmbracoDeploy.config on the different environment that has been set up, this will make sure that Deploy know which environment you are currently on when you visit one of them.
```

Comment:
```suggestion
When you have set up your environments in the `UmbracoDeploy.Config` the following `AppSetting` needs to be set on the different environments that you have set up with Umbraco Deploy:

```xml
<add key="Umbraco.Deploy.EnvironmentName" value="TheEnvironmentTypeHere" />
```

The value needs to be the **environment type** that has been set in the `UmbracoDeploy.config` on the different environments that have been set up. This will make sure that Deploy knows which environment you are currently in when you visit one of them.
```